### PR TITLE
Configure proof services for mobile

### PR DESCRIPTION
**Pull request by gabriel**
Fixes crash in proof checking on mobile (esp. for coyne), need to `SetServices`.

Also some other unrelated mobile fixes:
- Wrap favorites list saga in try catch
- Fix clicking on user in search going to the actual user
- Pro tip to skip gomobile init

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

**Comment by gabriel** [comment]
cc @maxtaco

File: go/libkb/services.go [hunk 1]
```diff
@@ -15,6 +15,10 @@ import (
 //=============================================================================
 
 func MakeProofChecker(c ExternalServicesCollector, l RemoteProofChainLink) (ProofChecker, ProofError) {
+	if c == nil {
+		return nil, NewProofError(keybase1.ProofStatus_UNKNOWN_TYPE,
```
Comment (maxtaco) [inline go/libkb/services.go]: When do you wind up in this case, on mobile?  You might be missing an initialization step...

**Comment by MarcoPolo** [comment]
js 👍

File: go/libkb/services.go (hunk 1 above)
Comment (gabriel) [inline go/libkb/services.go]: Yeah I added a SetServices() call in `bind/keybase.go`.

I added this to prevent a panic if services weren't configured.

Comment (gabriel) [inline go/libkb/services.go]: @max this ok?

Comment (gabriel) [inline go/libkb/services.go]: @maxtaco is this ok?

Comment (maxtaco) [inline go/libkb/services.go]: Sure, it's fine! I just wanted to make sure you saw how to avoid this error.

**Pull request merged**